openapi/src/test/scala/sttp/livestub/openapi/OpenapiParserSpec.scala

```diff
@@ -0,0 +1,15 @@
+package sttp.livestub.openapi
+
+import org.scalatest.flatspec.AnyFlatSpec
+import org.scalatest.matchers.should.Matchers
+
+class OpenapiParserSpec extends AnyFlatSpec with Matchers {
+  it should "parse correctly petstore specification" in {
+    val str = FileLoader.loadFile("petstore.yaml")
+    val openapi = YamlParser.parseFile(str)
+    openapi match {
+      case Left(value)  => fail(value)
+      case Right(value) => println(value)
```

Comment:
hm well ... would be nice to check at least something ;)

Reply:
for an e2e test, maybe you could start the livestub server on some port, and try running a request against the stub, seeing how it responds?

Reply:
Yes, definitely these tests need some work, I deliberately skip it because I wanted to get your opinion on the whole approach. 

Probably, I will do as you said